.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+---
+# desc: lint sanity check
+name: lint
+
+on:
+  push:
+  workflow_dispatch:
+
+
+jobs:
+  lint:
+    name: pre-commit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4
+      - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5
+
+      - uses: terraform-linters/setup-tflint@19a52fbac37dacb22a09518e4ef6ee234f2d4987 # v4
+        name: Setup TFLint
+        with:
+          tflint_version: v0.50.3
+
```

Comment:
I'd still prefer to pull this version from the `.tool-versions` via e.g. https://github.com/endorama/asdf-parse-tool-versions